modules/TimedText/resources/mw.TextSource.js

```diff
@@ -438,7 +438,9 @@
 					}
 					if( caption ){
 						// concatenate text lines to html text
-						caption['content'] = s.slice(2).join("<br>");
+						caption['content'] = s.slice(2).reduce(function(ac, current) {
```

Comment:
@omridevk is this an array slice and concat? I believe you can achieve the same result with array concat woth seperator